Sources/Tools/MaxComponent/plMiscComponents.cpp

```diff
@@ -158,13 +158,13 @@ ParamBlockDesc2 gInterestBk
     kInteresting,       _T("interesting"),      TYPE_STRING,        0, 0,   
         end,
     
-    kCamInterestRadius, _T("CamIntersestRadius"),   TYPE_FLOAT,     P_ANIMATABLE, 0,    
+    kCamInterestRadius, _T("CamInterestRadius"),   TYPE_FLOAT,     P_ANIMATABLE, 0,
```

Comment:
Looks like this would break old files

Reply:
I was afraid of that.

Reply:
I'm not sure this is correct.  From the [documentation](http://docs.autodesk.com/3DSMAX/16/ENU/3ds-Max-SDK-Programmer-Guide/cpp_ref/struct_param_def.html), it looks like the [ID](http://docs.autodesk.com/3DSMAX/16/ENU/3ds-Max-SDK-Programmer-Guide/cpp_ref/struct_param_def.html#a18f7550d7021a8da9ad427ac876718a6) `kCamInterestRadius` is the immutable identifier, whereas [the name](http://docs.autodesk.com/3DSMAX/16/ENU/3ds-Max-SDK-Programmer-Guide/cpp_ref/struct_param_def.html#adf74deb7ecf823157eef9de08c6eca6e) `CamInterestRadius` is simply an internal identifier presumably for runtime operation use.

What I can't tell is if this affects data serialization or not, but I would be inclined to think it may not.

Reply:
It shouldn't. Not sure what I was smoking above. The only thing I can think of that this would break is the text label. That's a key into the resource file you changed.